addons/l10n_ch/tests/test_onchange_l10n_ch_postal.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
```

Comment:
But it's not the way Odoo is organized. It might seem clearer this way, but its only because you are in a small module, with only a few tests. It's turning a 20 lines test into a 100 line file, imagine what more complex feature would look like if tested this way. Please rather adapt the original test to make it correct, and keep it in a single function. 

Also, asserting every time that your account number is right and your account type is correct seems a bit overkill to me. I think we can safely assume that field assignment works, and IBAN detection feature are not managed by this module either.